app/Http/Controllers/Api/ServersController.php

```
@@ -546,6 +555,21 @@ public function create_server(Request $request)
             ValidateServer::dispatch($server);
         }
 
+        // Start proxy if requested and not a build server
+        if ($request->start_proxy && ! $request->is_build_server) {
+            try {
+                $proxyShouldRun = CheckProxy::run($server, false);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Actions\Proxy\CheckProxy' in method 'create_server'. (undefined)

(StaticAccess)